openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -189,3 +189,15 @@ <h3 class="hd-6 section-title">${_("Course Handouts")}</h3>
 <%static:webpack entry="Enrollment">
     new CourseEnrollment('.enroll-btn', '${course_key | n, js_escaped_string}');
 </%static:webpack>
+
+<%static:require_module_async module_name="js/commerce/track_ecommerce_events" class_name="TrackECommerceEvents">
+
+  var PLSUpgradeLink = $(".PLS_button");
```

Comment:
Could we keep the class names consistent? e.g. all lowercase/no acronyms?

Reply:
Updated to expand PLS to personalized learner schedules